app/gui/src/ide/integration.rs

```diff
@@ -71,6 +72,59 @@ impl Model {
             });
         }
     }
+
+    /// Open a project by name. It makes two calls to Project Manager: one for listing projects and
+    /// a second one for opening the project.
+    pub fn open_project(&self, project_name: &str) {
+        let logger = self.logger.clone_ref();
+        let controller = self.controller.clone_ref();
+        let name = project_name.to_owned();
+        crate::executor::global::spawn(async move {
+            if let Ok(managing_api) = controller.manage_projects() {
+                match managing_api.list_projects().await {
+                    Ok(projects) => {
+                        let mut projects = projects.into_iter();
+                        let project = projects.find(|project| project.name.0 == name);
```

Comment:
I really don't like accessing `.0` in the code - it looks strange and tacky. Can we have comparison implemented for that or use any named method instead?